### DISCUSSION TITLE
Discord RPC

### DISCUSSION THREAD

**Discussion by 2MPP** [jeffvli/feishin#762] [General]
Is they anyway to display the songs cover icon on the discord rpc?

so instead of this
![image](https://github.com/user-attachments/assets/85bf6cd9-289f-4cbe-9431-47b1df8bcb62)
it will display this
![image](https://github.com/user-attachments/assets/653dd007-85a3-4acc-a9f5-2f121d6dd02f)
as the icon?

**Comment by iiPythonx** [comment]
This is being discussed a lot in https://github.com/jeffvli/feishin/issues/341, right now the problem is you would either need your server to be public (so discord can load the art, [which on navidrome is a security risk](https://github.com/jeffvli/feishin/pull/515#issuecomment-1950706808)), or Feishin would have to upload art to a service like Imgur.

The bottom line is no, Feishin doesn't support that right now (unless you're using Jellyfin **and** it's public, in which case you can follow https://github.com/jeffvli/feishin/issues/341#issuecomment-2179602461).

**Reply by 2MPP** [reply]
Ah okay its sad that it requires sending credentials to grab the image

**Reply by 2MPP** [reply]
I attempted to do the the jellyfin way [#341 comment](https://github.com/jeffvli/feishin/issues/341#issuecomment-2179602461) suggested but that didnt seem to work the server is open publicly

**Reply by iiPythonx** [reply]
I can't help you much there since I don't use Jellyfin.  
If you would like though (and you don't use MacOS), you could try running a [version of my fork](https://github.com/iiPythonx/feishin/releases) which has a free image proxy hosted by me built into it (Settings -> Tweaks -> Image proxy type -> Pizza).

**Reply by 2MPP** [reply]
I will give it a try and update u on it

**Reply by 2MPP** [reply]
Hi this dosnt even seem to give me a discord RPC on my profile
![image](https://github.com/user-attachments/assets/51a47f34-5494-436e-99ee-335bf0404715)
![image](https://github.com/user-attachments/assets/d7879c44-203a-4465-9542-d7b2cea0cb74)
What is ur discord so we can talk on there

**Reply by iiPythonx** [reply]
`@iipython`, shoot me a dm

**Comment by HamonTeacher** [comment]
> Is they anyway to display the songs cover icon on the discord rpc?
> 
> ![image](https://private-user-images.githubusercontent.com/37121247/370885310-85bf6cd9-289f-4cbe-9431-47b1df8bcb62.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc2OTA1MDUsIm5iZiI6MTcyNzY5MDIwNSwicGF0aCI6Ii8zNzEyMTI0Ny8zNzA4ODUzMTAtODViZjZjZDktMjg5Zi00Y2JlLTk0MzEtNDdiMWRmOGJjYjYyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTMwVDA5NTY0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJiNWQ1ODI3ZWRlMjQ5ZmU3ODNkMzgwNzU3YWZhZTUyNDk3MTAxMDYzMzRiODA1YjJiZDgwNDgyYTVmMmMyNjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.t6QKZrf0W5bxSvV3lUXJ7ColUOQNKP1wS3Qchx9BH7Y)

I don't even have that display, where do I dig?
